### ISSUE TITLE
Fade: Using Collapse css properties

### ISSUE THREAD

**Issue by ahabbaba9** [react-bootstrap/react-bootstrap#1088]
I've got a couple of components that I'm rendering and one is `Fade`.

I've essentially taken the example `Fade` code from the [docs](http://react-bootstrap.github.io/components.html#utilities). When it's rendered, it shows the `collapse` class instead of `fade`:

```
<div style="" aria-expanded="true" class="collapse in" data-reactid=".0.2.0">
<div class="well" data-reactid=".0.2.0.0">
Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident.
</div>
</div>
```

Wondering what could be going wrong here?

Thanks!

<b>UPDATE</b>

I've copied and pasted the example code from the docs (same link as above) and it's still collapsing the component, not fading it in/out.

**Comment by taion** [comment]
Do you have a repro for this? You can see that the example in the docs is applying the correct styling. Are you sure you're importing the correct component?

**Comment by ahabbaba9** [comment]
Sorry for the late response. Yes I know I'm imported the correct component (Fade)

**Comment by taion** [comment]
Right, it was actually a bug, fixed by #1159.

**Comment by ahabbaba9** [comment]
Oh I see, thanks!
